assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -721,6 +721,10 @@ function initializeDiscourseCalendar(api) {
     (post.calendar_details || []).forEach((detail) => {
       switch (detail.type) {
         case "grouped":
+          if (fullDay && detail.timezone) {
+            let from = moment.tz(detail.from, detail.timezone);
+            detail.from = from.format("YYYY-MM-DD");
```

Comment:
```suggestion
            detail.from = moment
              .tz(detail.from, detail.timezone)
              .format("YYYY-MM-DD");
```

Is this legal? Also curious why the `let`